### PR TITLE
feat(ngrx-chat): :sparkles: UI follow the Bitovi design system

### PR DESCRIPTION
**Pull request by ilyassFouih**

### CODE REVIEW

File: src/app/chat/chat-header/chat-header.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@ <h1>List of received messages</h1>
 <!-- unedited form -->
 <div class="wrapper" [ngClass]="{ 'wrapper-center': !editUsername }">
   <ng-container *ngIf="!editUsername">
-    <h4>Logged in as : {{ username$ | async }}</h4>
+    <h2>Logged in as : {{ username$ | async }}</h2>
```
Comment (fabioemoutinho) [inline src/app/chat/chat-header/chat-header.component.html:6]: nice catch for a11y

File: src/app/chat/message/message.component.ts [hunk 2]
```diff
@@ -10,9 +11,17 @@ import { UserDataPipe } from './user-data.pipe';
...
-  @Input() authenticatedUsername!: string | null;
-  protected readonly MessageStatus = MessageStatus;
+  @Input() authUserName!: string | null;
+  @Input() authUserId!: string | null;
+
+  isExternalMessage!: boolean;
+  protected PENDING = MessageStatus.PENDING;
+  protected ONLINE = UserStatus.ONLINE;
+
+  ngOnChanges({ message, authUserId }: SimpleChanges): void {
```
Comment (fabioemoutinho) [inline src/app/chat/message/message.component.ts]: can we revert the changes here? While I think it adds to readability in the template, I don't expect users to do it this way in the training

File: src/app/chat/message/message.component.ts [hunk 3]
```diff
@@ -1,27 +1,21 @@
...
-  imports: [CommonModule, UserDataPipe],
+  imports: [CommonModule],
 })
-export class MessageComponent implements OnChanges {
+export class MessageComponent {
   @Input() message!: Message;
-  @Input() authUserName!: string | null;
-  @Input() authUserId!: string | null;
+  @Input() connectedUserIsSender!: boolean;
+  @Input() userStatus!: UserStatus | null | undefined;
```
Comment (fabioemoutinho) [inline src/app/chat/message/message.component.ts:16]: why do we need to account for undefined here? 🤔

Comment (ilyassFouih) [reply src/app/chat/message/message.component.ts:16]: - For `userStatus` it's marked as optional [here](https://github.com/bitovi/angular-ngrx-chat/blob/60c35215c6b79f91b896a04cbe8b5b191ed338cd/src/app/chat/chat-store/chat.model.ts#L22).
- And for `userStatus` could be undefined [here](https://github.com/ilyassFouih/angular-ngrx-chat/blob/206d0b7f8a0c5c54dab41750956c48e404c2a8c4/src/app/chat/chat.component.html#L11)

File: src/app/chat/message/user-data.pipe.ts [hunk 4]
```diff
@@ -4,10 +4,10 @@ import { User } from '../chat-store/chat.model';
 import { ChatStore } from '../chat-store/chat.store';
 
 @Pipe({
-  name: 'userData',
+  name: 'getUserById',
   standalone: true,
 })
-export class UserDataPipe implements PipeTransform {
+export class getUserById implements PipeTransform {
```
Comment (fabioemoutinho) [inline src/app/chat/message/user-data.pipe.ts]: classes should start with an Uppercase letter. I don't like the get prefix for a pipe tho, maybe UserById is better?

Comment (ilyassFouih) [reply src/app/chat/message/user-data.pipe.ts]: @fabioemoutinho Nice catch for the Class name, Thank you. 
For the prefix both are fine with me.

**Review by fabioemoutinho — APPROVED** [review]
